.github/workflows/ci.yml

```diff
@@ -374,6 +380,32 @@ jobs:
           pipenv run pylint --disable=all --enable=import-error garage"
 
 
+  deploy_to_PYPI:
+    name: Deploy to PYPI
+    runs-on: ubuntu-latest
+    needs: [normal_test, large_test, mujoco_test, mujoco_test_long]
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Please make sure this *only* matches `refs/tags/` which correspond with the tags regex above!

Reply:
unfortunately there's no way to do regex matching an if statement. I'm assuming that if a tag runs, it will be one that matches the trigger, which is a correct regex